msteams-platform/concepts/build-and-test/teams-developer-portal.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Manage your apps with the Developer Portal for Microsoft Teams
+description: Learn manage your apps using the Developer Portal for Microsoft Teams
+keywords: getting started developer portal teams
+localization_priority: Normal
+ms.topic: overview
+---
+
+# Developer Portal Overview
+
+The Developer Portal enables app creators to register apps with the Teams app service and publish to an organization's app catalog, or the Teams store. You can invite colleagues in your organization to collaborate on an app, configure runtime environment, and more.
```

Comment:
RE: "_The Developer Portal enables app creators to register apps with the Teams app service **and publish to an organization's app catalog, or the Teams store_**"

We should not mention publish in the value proposition statement for Dev Portal. These actions are less seldom used and not the primary purpose of DP.

How about:

"**Easily register, configure and manage your Teams apps from the Developer Portal.**"